app/views/front_view/index.html.erb

```diff
@@ -18,13 +34,36 @@
     document.getElementById("current_date").innerHTML = day + "/" + month + "/" + year;
   </script>
 
-  <form id="form_style"> 
-    <label for="employee_num">Numero de trabajador:</label> 
-    <input type="text" id="employee_num" name="employee_num"> <br> <br>
 
-    <label for="employee_private_num">Numero de trabajador secreto:</label> 
-    <input type="text" id="employee_private_num" name="employee_private_num"><br><br>
+  <div class="row justify-content-center text-center ">
+    <%= form_with url: front_view_index_path, method: :post do |form| %>
+        <div class="mb-2 ">
+          <div>
+            <%= form.label :"employee number:" %>
+          </div>
+          <%= form.number_field :employee_num %>
+        </div>
+
+        <div class="mb-2 pt-1">
+          <div>
+            <%= form.label :"private number:" %>
+          </div>
+          <%= form.password_field :employee_private_num %>
+        </div>
+
+        <div class="message">
+          <% flash.each do |key, value| %>
+            <div class="message__content <%= key %>">
+              <%= value %>
+            </div>
+          <% end %>
+        </div>
+
+        <div class="check__buttons_container pt-1">
+          <%= form.submit "Check in", name:"check", value:"in", class: "check__buttons_container__in " %>
+          <%= form.submit "Check off", name:"check", value:"off",  class: "check__buttons_container__off " %>
+        </div>
+      <% end %>
+  </div>
+</div>
 
-    <button>Check-in</button>
-  </form>
-</div>
```

Comment:
Sorry maybe I didn't see it, but this one also needs an empty line at the end.

Reply:
there´s an empy line at the end of that file, don't know why github doesn't show it.